VTEX - Master Data API - v2.json

```diff
@@ -26,8 +26,8 @@
+                "summary": "Create new document with random ID",
```

Comment:
I feel ambiguous about the word "random" and its role in this summary. For the description, I would go with something like "automatically generated by VTEX".

The summary is harder, but maybe stick to the old title? I mean, the fact that the ID is random is a small detail that most people won't care about. Maybe putting it in the title will be confusing, like "should I be worried about the doc ID?". Maybe adding a callout and linking to the other endpoint is good enough?